docs/SHAPCLI.md

```diff
@@ -49,3 +49,18 @@ Using the `-h` flag in each option will output a new help output. For example:
 ```
 shapcli hana -h
 ```
+
+### Running commands in remote nodes
+
+The commands can be executed in remote nodes too. For that the `-r` or `--remote` flag have to be
+used (or adding the `remote` entry in the configuration file [the `-r` flag has priority over the
```

Comment:
regarding this `-r` precedence. This seems to me a design concern which should not be part of this doc.

When we say : `e [the `-r` flag has priority over the configuration`,

we should not state this here, We should state in a global/central place that all the CLI flags have precedence/overwrite the config file data ( this explained less tech).

Reply:
But this is not the case right. 
The code only applies this logic to the `-r` flag. The other arguments are loaded from the configuration if this is provided.
I implemented this way, because the `remote` flag is quite special, and I see many use cases where we will run the same command in the local and remote nodes, so only this flag will be added.

Anyway, we can discuss if this should be applied to all flags (actually I like the idea).

We could open a issue here and work in this feature in the future.

Reply:
OK imho we should apply this to all options because otherwise we have a special case which is in long-term tricky to maintain and prone to errors. Just refer here the issue and feel free to resolve thx

Reply:
ok https://github.com/SUSE/shaptools/issues/38